source/includes/gcp/_k8_releases.md

```diff
@@ -319,7 +319,31 @@ Attributes | &nbsp;
 `version`<br/>*string* | The revision of the release
 `namespace`<br/>*string* | The namespace to which the release is installed
 
+<!-- ROLLBACK RELEASE -->
+### Rollback release to previous revision
```

Comment:
This is missing a sub level again. Please check the other one from gcp to conform to it

Reply:
Made sure sub-levels for releases in gcp are all correct in https://github.com/cloudops/cloudmc-api-docs/pull/86/commits/7cee492164ec7e3e4994afdc992e4ad5a4677f19